examples/workflow-template/top-level-tmpl-ref.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: workflow-template-hello-world-
+spec:
+  entrypoint: whalesay-template
+  arguments:
+    parameters:
+      - name: message
+        value: "hello world"
+  workflowTemplateRef:
+    name: workflow-template-whalesay-template
+
```

Comment:
Can we have two examples? One that utilizes arguments/entrypoint and one that doesn't. 

```yaml
apiVersion: argoproj.io/v1alpha1
kind: Workflow
metadata:
  generateName: workflow-template-hello-world-
spec:
  workflowTemplateRef:
    name: workflow-template-whalesay-template
```